packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -33,14 +33,26 @@ contract L1Block is ISemver {
     bytes32 public batcherHash;
 
     /// @notice The overhead value applied to the L1 portion of the transaction fee.
+    /// @custom:legacy
     uint256 public l1FeeOverhead;
 
     /// @notice The scalar value applied to the L1 portion of the transaction fee.
+    /// @custom:legacy
     uint256 public l1FeeScalar;
 
-    /// @custom:semver 1.1.0
-    string public constant version = "1.1.0";
+    /// @notice The latest L1 blob basefee.
+    uint256 public blobBasefee;
 
+    /// @notice The scalar value applied to the L1 blob base fee portion of the blob-capable L1 cost func
+    uint32 public blobBasefeeScalar;
+
+    /// @notice The scalar value applied to the L1 base fee portion of the blob-capable L1 cost func
+    uint32 public basefeeScalar;
```

Comment:
Since we're evidently trying very hard to reduce gas costs here, we could save two sstores by taking advantage of the fact that the `sequenceNumber` is only 8 bytes in a 32 byte slot, and pack these values into it.

If we get the `sstore`s right, then the `public` getters should still work just by moving these variable declarations up.

The one downside I can see is that it it might feel like an awkward order to be declaring these variables in.

Reply:
it does seem safe (but slightly awkward) to do this. I don't have strong preference here though. WDYT @tynes ?

Reply:
Decided I liked this idea and went ahead and implemented it. I think it's nicer this way. PTAL?